custom_udf/build_custom_udf.sh

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env bash
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+##############################################################
+# This script is used to compile UDF 
+# Usage:
+#    sh build-custom-udf.sh            build udf without clean.
+#    sh build-custom-udf.sh -clean     clean previous output and build.
+#
+##############################################################
+
+set -eo pipefail
+
+ROOT=`dirname "$0"`
+ROOT=`cd "$ROOT"; pwd`
+
+export DORIS_HOME=$(dirname "$PWD")
+echo ${DORIS_HOME}
+export CUSTOM_UDF_HOME=${ROOT}
+
+. ${DORIS_HOME}/env.sh
+
+# build thirdparty libraries if necessary
```

Comment:
no need to build thirdparty in this script.
This should be done `build.sh`.
Just make this script simple.